src/test/java/io/spine/helloworld/server/hello/HelloContextTest.java

```diff
@@ -11,13 +12,11 @@
 import static io.spine.testing.TestValues.randomString;
 
 @DisplayName("Hello Context should")
-class HelloContextTest {
+class HelloContextTest extends ContextAwareTest {
```

Comment:
Can we make the class `final`?

Reply:
We probably can, but this is not customary in JUnit.